docker-compose.env

```diff
@@ -59,3 +59,12 @@ RUN_STATE_BATCH_SUBMITTER=true
 
 L1_NODE_WEB3_URL=http://l1_chain:9545
 L2_NODE_WEB3_URL=http://geth_l2:8545
+
+######### L1->L2 / L2->L1 Tests #########
+L1_USER_PRIVATE_KEY="0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a"
+L2_USER_PRIVATE_KEY="0x570afe67c36ac0ae058df4b5c28b49e1381ffbce761866750f0cc41918d0d383"
+L1_MESSENGER_ADDRESS="0x6418E5Da52A3d7543d393ADD3Fa98B0795d27736"
+L2_MESSENGER_ADDRESS="0x4200000000000000000000000000000000000007"
+
+# Address Manager
+ETH1_ADDRESS_RESOLVER_ADDRESS=0x3e4CFaa8730092552d9425575E49bB542e329981
```

Comment:
This is already supplied via https://github.com/ethereum-optimism/docker/blob/f0db880135371ea61cf747ee9de6062f20ea99f5/integration-tests/wait-for-l1-and-l2-and-contract-deployment.sh#L55